doc/melange.rst

```diff
@@ -0,0 +1,66 @@
+.. _melange_main:
+
+***********************************
+JavaScript Compilation With Melange
+***********************************
+
+`Melange <https://github.com/melange-re/melange>`_ is a compiler from OCaml to
+JavaScript. Unlike Js_of_ocaml, Melange works by translating OCaml compiler
```

Comment:
```suggestion
JavaScript. Unlike js_of_ocaml, Melange works by translating OCaml compiler
```
Unclear for the capitalization of this. My notes say sentence case (at beginning of sentences and in titles, but not in middle of sentences), but should the J always be capitalized?

Reply:
I think you are right, I checked the official documentation and it seems to follow the casing rules you mention, e.g.: http://ocsigen.org/js_of_ocaml/latest/manual/separate-compilation.

Reply:
Great! Thanks for confirming that!